string bin_name
float32 x_coordinate
float32 y_coordinate
bool isFull
string to_attach_name
string picker_name
